integration-fe/src/sagas/getInstance.js

```diff
@@ -0,0 +1,11 @@
+import { call, put } from 'redux-saga/effects';
+import { getInstance as activeInstance } from './apiCalls';
```

Comment:
this saga gets the "active instance". This handles with an in-memory object to create, edit or remove an instance.